src/patternfly/components/Select/select-single.hbs

```diff
@@ -4,22 +4,21 @@
   {{#if select--attribute}}
     {{{select--attribute}}}
   {{/if}}>
-  <li><button class="pf-c-select__menu-item" aria-pressed="false">January</button></li>
-  <li><button class="pf-c-select__menu-item" aria-pressed="false">February</button></li>
-  <li><button class="pf-c-select__menu-item" aria-pressed="false">March</button></li>
+  <li><button class="pf-c-select__menu-item" aria-pressed="false">Running</button></li>
```

Comment:
I think these changes are OK for now but we need to make sure this select isn't called anywhere else and the change has introduced a select with odd choices in another example/demo. Also can you update the toggle text in the select component examples? It says "filter by birth month" with these new options currently.

Opened an issue to handle this better here - https://github.com/patternfly/patternfly-next/issues/2300

Reply:
Thanks for opening the issue. Will check other examples and update the toggle text